layouts/_default/page-landing.html

```diff
@@ -2,80 +2,73 @@
   font-sans
 {{ end }}
 {{ define "html-class" -}}
-  scroll-smooth
+  scroll-smooth scroll-pt-10 lg:scroll-pt-40
 {{- end }}
 
 {{ define "main" }}
-  {{ with .Page.RenderString (.Param "takeover") }}
-    {{ . }}
-  {{ else }}
-    <header class="bg-white px-5 pb-4 pt-8 text-black">
-      <div class="mx-auto max-w-screen-lg">
-        {{ partial "tw/breadcrumbs.html" . }}
-      </div>
-      <div class="mx-auto w-full max-w-2xl">
+  <section class="bg-white pb-6">
+    {{ if .Page.RenderString (.Param "takeover") }}
+      {{ .Page.RenderString (.Param "takeover") }}
+    {{ else }}
+      <div
+        class="mx-auto w-full max-w-screen-xl flex-wrap gap-4 pt-16 md:gap-x-6 lg:gap-x-12"
+      >
         <h1
-          class="font-sans text-3xl font-bold leading-tight text-black sm:text-4xl sm:leading-tight lg:text-5xl lg:leading-tight"
+          class="{{ with .Page.RenderString (.Param "sidebar") }}
```

Comment:
I think this can be just `{{ if .Param "sidebar" }}max-w-prose{{ else }}md:w-7/12{{ end }}`.